templates/taxi/driver_list.html

```diff
@@ -1,7 +1,10 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Driver List
+    <h1>Driver List 
+      <a style="float: right" href="{% url 'taxi:driver-create' %}">
+      <button class="btn btn-primary">Create driver</button>
+      </a>
```

Comment:
Placing a `<button>` element inside an `<a>` tag is not semantically correct in HTML. Consider using a `<button>` element with a `type="button"` and an `onclick` attribute to handle navigation, or style the `<a>` tag directly as a button.